contrib/opentimelineio_contrib/adapters/ale.py

```diff
@@ -134,12 +134,30 @@ def _parse_data_line(line, columns, fps):
 
             if cdl:
                 del metadata['ASC_SOP']
+                asc_sop = cdl.get('asc_sop')
+                if asc_sop:
+                    line += "\t*ASC_SOP ({} {} {}) ({} {} {}) ({} {} {})\t".format(
+                        asc_sop['slope'][0],
+                        asc_sop['slope'][1],
+                        asc_sop['slope'][2],
+                        asc_sop['offset'][0],
+                        asc_sop['offset'][1],
+                        asc_sop['offset'][2],
+                        asc_sop['power'][0],
+                        asc_sop['power'][1],
+                        asc_sop['power'][2]
+                    )
 
         if metadata.get('ASC_SAT'):
             try:
                 asc_sat_value = float(metadata['ASC_SAT'])
                 cdl.update(asc_sat=asc_sat_value)
                 del metadata['ASC_SAT']
+                asc_sat = cdl.get('asc_sat')
+                if asc_sat:
+                    line += "\t*ASC_SAT {}\t".format(
+                        asc_sat
+                    )
```

Comment:
I think this update code is in the reader, not the writer. Also, the string looks like it's in a format meant for EDL?